src/controller/CHIPDeviceController.h

```diff
@@ -622,6 +622,15 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
        If no device is currently being paired, this value will be kNumMaxPairedDevices.  */
     uint16_t mDeviceBeingPaired;
 
+    enum CertificateChainType : uint16_t
+    {
+        kUnknown = 0,
+        kDAC     = 1,
+        kPAI     = 2,
+    };
```

Comment:
This is an enum that should live in the cluster definition/object. If we cannot have it there, then let's put it temporarily with the DAC verifier code, rather than deeply nested in this class. This is not knowledge whose "source of truth" should be DeviceCommissioner

Reply:
will move to DAC verifier